src/Console/Helper/TaskRunnerHelper.php

```diff
@@ -97,55 +98,63 @@ private function registerEventListeners(OutputInterface $output)
 
     /**
      * @param OutputInterface $output
-     * @param array $errorMessages
-     * @param array $warnings
+     * @param array           $errorMessages
+     *
+     * @return int
      */
-    private function printErrorMessages(OutputInterface $output, array $errorMessages, array $warnings)
+    private function returnErrorMessages(OutputInterface $output, array $errorMessages, array $warnings)
```

Comment:
Do you really think this is a better name? This method does not return error messages, it returns an error code. I believe *printErrorMessages* is still the correct name for this method, and moreover, the decision about the code should be left to the caller.